susemanager-utils/testing/docker/scripts/test_cobbler.sh

```diff
@@ -22,7 +22,27 @@ cp /root/sample.ks /var/lib/cobbler/kickstarts/sample.ks
 # start cobbler daemon
 cobblerd
 
+# Configure DHCP
+sed -i 's/DHCPD_INTERFACE=""/DHCPD_INTERFACE="ANY"/' /etc/sysconfig/dhcpd
+echo "subnet 172.17.0.0 netmask 255.255.255.0 {}"  >> /etc/dhcpd.conf
+
+# Configure PAM
+useradd -p $(perl -e 'print crypt("test", "password")') test
+
+sh /root/cobbler/setup-supervisor.sh
+
 # execute the tests
 
 cd /usr/share/cobbler/tests
-pytest --junitxml=/reports/cobbler.xml
+
+pytest --junitxml=/reports/cobbler.xml \
+    --deselect=tftpgen_test.py::test_copy_single_distro_file \
+    --deselect=utils_test.py::test_is_safe_to_hardlink[/etc/os-release-/tmp-False] \
+    --deselect=utils_test.py::test_get_file_device_path \
+    --deselect=utils_test.py::test_local_get_cobbler_api_url \
+    --deselect=xmlrpcapi/image_test.py::TestImage::test_copy_image \
+    --deselect=xmlrpcapi/image_test.py::TestImage::test_find_image \
+    --deselect=xmlrpcapi/image_test.py::TestImage::test_get_image \
+    --deselect=xmlrpcapi/image_test.py::TestImage::test_remove_image \
+    --deselect=xmlrpcapi/image_test.py::TestImage::test_rename_image \
+    --deselect=xmlrpcapi/non_object_calls_test.py::test_get_item_resolved_value[interfaces-system-expected_result4-expected_exception4]
```

Comment:
The calls behind these tests are all used in our product. Is there a reason why they are failing that can be fixed?

Reply:
These failing tests can be fixed, sure. We need to adjust some tests expectations and do some small fixes here and there, as mentioned on the description.

Let's do something, since I expected we can do and push these fixes for the Cobbler tests soon, I'm removing the full list of deselected tests from here, making our Jenkins jobs to be still failing for some time until we push those fixes. So we can get this PR in already.